core/templates/pages/collection-player-page/collection-local-nav/collection-local-nav.component.html

```diff
@@ -0,0 +1,15 @@
+<div>
+  <ul class="nav navbar-nav oppia-navbar-nav float-right oppia-navbar-pencil-icon">
+    <li *ngIf="canEdit" class="nav-item">
+      <a [href]="'/collection_editor/create/' + collectionId" ngbTooltip="Edit" placement="bottom" target="_blank" rel="noopener" class="nav-link">
+        <i class="material-icons">&#xE254;</i>
+        <span class="oppia-icon-accessibility-label">Edit</span>
+      </a>
+    </li>
+  </ul>
+</div>
+<style>
+  .collection-local-nav .oppia-navbar-pencil-icon {
```

Comment:
Where is this `.collection-local-nav` class defined?

Reply:
removed it!